### PR TITLE
Extract css to authentication.css

### PR DESCRIPTION
**Pull request by m-pereira**
partially resolves: https://github.com/jonmchan/pseudo_kiosk/issues/3

### Description:
This PR is extracting the css from app/views/pseudo_kiosk/authentication/unlock.html.erb to app/assets/stylesheets/pseudo_kiosk/authentication.css

### CODE REVIEW

**Comment by m-pereira** [comment]
@jonmchan can you help me? I don't know why it's not working, once it has the ```<%= stylesheet_link_tag    "pseudo_kiosk/application", media: "all" %>``` in ```app/views/pseudo_kiosk/layouts/application.html.erb``` and application is requiring the tree.

File: app/views/pseudo_kiosk/authentication/unlock.html.erb [hunk 1]
```diff
@@ -1,260 +1,105 @@
...
+    </script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/prefixfree/1.0.7/prefixfree.min.js"></script>
+    <script>
+      if (document.location.search.match(/type=embed/gi)) {
+        window.parent.postMessage("resize", "*");
+      }
+    </script>
+  </head>
+  <body translate="no">
+    <div id="explanation_window" class="center-xy <%= params[:failed] ? 'explanation_window__failed' : 'explanation_window' %>">
```
Comment (jonmchan) [inline app/views/pseudo_kiosk/authentication/unlock.html.erb]: Excellent job! This is much cleaner than I had it before!

**Comment by jonmchan** [comment]
Please fix the style coloring for the fail state.

The red no longer shows and the fonts aren't there anymore. It would seem that some part of the styles are not being applied? 

Screen Shot 2019-11-06 at 4 03 12 PM

Original was:
Screen Shot 2019-11-06 at 4 05 06 PM

File: app/views/pseudo_kiosk/authentication/unlock.html.erb [hunk 2]
```diff
@@ -1,260 +1,105 @@
...
+      <h2>Please pass the device back to the operator</h2>
+      <h1>Slide to Unlock</h1>
+      <input type="range" value="0" class="pullee" />
+    </div>
+    <% # This is bad... I am having a problem using the named routes within the rails engine plugin %>
+    <%= form_tag 'process_submit', authenticity_token: false do %>
+      <%= hidden_field_tag :authenticity_token, form_authenticity_token -%>
+      <div id="pass_window" class="center-xy <%= params[:failed] ? 'pass_window__failed' : 'pass_window' %>">
+        <h1>Passcode:</h1>
+        <div id="invalid" class="<%= params[:failed] ? 'invalid_failed' : 'invalid' %>">Invalid Passcode! Please try again.</div>
```
Comment (jonmchan) [inline app/views/pseudo_kiosk/authentication/unlock.html.erb]: the css issue seems like a typo here? In the style sheet, it is `invalid__failed`, while this is `invalid_failed`.

Comment (m-pereira) [reply app/views/pseudo_kiosk/authentication/unlock.html.erb]: Oh, good catch! I'm gonna fix that!

**Comment by m-pereira** [comment]
@jonmchan I think that is good enough for the first extraction. WDYT?

**Comment by jonmchan** [comment]
Yes, great work! I'll cut a new release later. Thanks for your contribution @m-pereira.

**Pull request merged**